src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,42 @@
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+interface PeopleListProps {
+  peopleList: Person[];
+}
+
+export const PeopleTable: React.FC<PeopleListProps> = ({ peopleList }) => {
+  const getParent = (name: string) => {
+    return peopleList.find(person => person.name === name);
+  };
```

Comment:
You can prepare the data in the parent component so `PeopleTable` will be a representation component and doesn't have any logic, just render UI
https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0
https://www.freecodecamp.org/news/separation-of-concerns-react-container-and-presentational-components/